EIPS/eip-1052.md

```diff
@@ -1,7 +1,8 @@
 ---
 eip: 1052
 title: EXTCODEHASH opcode
-author: Nick Johnson <arachnid@notdot.net>, Paweł Bylica <pawel@ethereum.org>
+description: Returns the keccak256 hash of a contract's code
+author: Nick Johnson (@arachnid), Paweł Bylica (@chfast)
```

Comment:
This is a final proposal, so it isn't important to update the author line. No one has automatic merge permissions anyway.

```suggestion
author: Nick Johnson <arachnid@notdot.net>, Paweł Bylica <pawel@ethereum.org>
```